migrate/migrations/20220317153438-alert-metrics-add-closed-at.sql

```diff
@@ -0,0 +1,18 @@
+-- +migrate Up
+
+UPDATE engine_processing_versions
+SET "version" = 2
+WHERE type_id = 'metrics';
+
+ALTER TABLE alert_metrics ADD COLUMN closed_at TIMESTAMP WITH TIME ZONE;
```

Comment:
We should make this not-null and truncate the table so we recalculate existing data